src/Endpoint.js

```diff
@@ -38,6 +38,91 @@ class Schema extends Component {
   }
 }
 
+const callParser = (res) => {
```

Comment:
Can you add comments here? The recursive code gets a bit difficult to read through, comments for each map block will make it easier to parse for new users to the library who want to contribute.